### ISSUE TITLE
read function for Wavefront Obj format

### ISSUE THREAD

**Issue by 9prady9** [9prady9/MeshIO#25]

**Comment by 9prady9** [comment]
@acrlakshman Does local changes include reading OBJs too ?

**Comment by acrlakshman** [comment]
Yes they do. But I am rewriting both of them because the tinyobj library I was depending on has changed since I started. Now at this point I wanted to check with you the following 2 things
1. Should be distribute work between eachother for read and write part just like we did with STL. Since I am doing a fresh start again?
2. Should be continue to depend on tinyobj, since that is well established at this point? I am inclined towards depending on that.

**Comment by 9prady9** [comment]
If you have it written already, there is no point me taking over it from you. I think it might be faster if you get it finished. I can start reading about glTF.

About tinyobj reader, is it that hard to write our own obj reader ? I personally don't want to have any dependency unless absolutely necessary. That way, meshio headers are not dependent on anything else. Helps us learn things on the way too. Also, I wanted to discuss something with you. Perhaps, direct message me.

**Comment by acrlakshman** [comment]
Please find `read_write_OBJ` branch here: https://github.com/acrlakshman/MeshIO/tree/read_write_OBJ
